From 613057152c4f6caba22a15ed2ff0aeb9c4ce6a83 Mon Sep 17 00:00:00 2001
From: Jakub Martisko <jamartis@redhat.com>
Date: Wed, 5 Feb 2020 15:50:15 +0100
Subject: [PATCH] enable the -b option on all platforms

---
 sed/sed.c | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/sed/sed.c b/sed/sed.c
index f75e4b6..1ca5839 100644
--- a/sed/sed.c
+++ b/sed/sed.c
@@ -174,7 +174,12 @@ Usage: %s [OPTION]... {script-only-if-no-other-script} [input-file]...\n\
   fprintf (out, _("  -b, --binary\n\
                  open files in binary mode (CR+LFs are not" \
                  " processed specially)\n"));
+#else
+  fprintf (out, _("  -b, --binary\n\
+                 does nothing; for compatibility with WIN32/CYGWIN/MSDOS/EMX \n\
+                 (open files in binary mode; CR+LF are not processed specially)\n" ));
 #endif
+
   fprintf (out, _("  -l N, --line-length=N\n\
                  specify the desired line-wrap length for the `l' command\n"));
   fprintf (out, _("  --posix\n\
-- 
2.24.1

